js/core.js

```diff
@@ -1145,7 +1145,7 @@ async function parsePhraseBook(s, loadSketch) {
             sketch = importedSketches[o.name];
         } else {
             let snap = await loadSketch.download(`sketch/${o.name}`);
-            sketch = Object.assign({}, snap.val());
+            sketch = Object.assign({}, snap);
```

Comment:
This Object.assign call is now unnecessary, since we don't need to copy anything anymore. Also, the name `snap` comes from Firebase (short for "snapshot") so the variable should probably be named differently.

I would just do `const sketch = await loadSketch(o.name);`